vpm-flexible-gallery.php

```diff
@@ -97,13 +97,18 @@ function vpm_custom_gallery_output( $output, $attr, $instance = null ) {
 		$img = wp_get_attachment_image_src( $id, $size );
 		$src = apply_filters( 'vpm_gallery_image_src', $img_full[0], $id, $size );
 
+		// Fetch image caption
+		$caption = $attachment->post_excerpt;
```

Comment:
If this is empty, it still renders an empty `<dd class="wp-caption-text gallery-caption"></dd>` element. Can we shuffle our logic or add more logic to ensure these empty tags don't output?